clang-tools-extra/clang-tidy/modernize/UseNodiscardCheck.cpp

```diff
@@ -84,33 +82,32 @@ void UseNodiscardCheck::storeOptions(ClangTidyOptions::OptionMap &Opts) {
 }
 
 void UseNodiscardCheck::registerMatchers(MatchFinder *Finder) {
-  auto FunctionObj =
-      cxxRecordDecl(hasAnyName("::std::function", "::boost::function"));
-
   // Find all non-void const methods which have not already been marked to
```

Comment:
Update comment. What does it have to do with the method being const or not? It looks like nothing...

Reply:
Yes, good catch, this comment  is leftover from the original check, which did trigger for const methods only.